application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -59,6 +59,21 @@ analytics.mostViewedPages.parameter.label.startDate=Start time of the date inter
 rendering.macro.mostViewedPages.description=When visitors search on your website, they are looking for a particular page, content, product, or service. This report lists the pages that were clicked the most after an internal search.
 rendering.macro.mostViewedPages.name=Most viewed pages
 
+## Exit Pages
+analytics.exitPages.header.actions=Actions
+analytics.exitPages.header.exitRate.hint=The percentage of visits that left the website after viewing this page.
+analytics.exitPages.header.exitRate=Exit rate
+analytics.exitPages.header.exitNbVisits.hint=Number of visits that ended on this page.
+analytics.exitPages.header.exitNbVisits=Exits
+analytics.exitPages.header.pageTitle=Title
+analytics.exitPages.header.visits.hint=The number of visits that included this page. If a page was viewed multiple times during one visit, it is only counted once.
+analytics.exitPages.header.visits=Unique visitors
+analytics.exitPages.header.year=Year
+analytics.exitPages.parameter.label.endDate=End time of the date interval used for this macro
+analytics.exitPages.parameter.label.startDate=Start time of the date interval used for this macro
+rendering.macro.exitPages.description=This report contains information about the exit pages that occurred during the specified period. An exit page is the last page that a user views during their visit. The exit URLs are displayed as a folder structure. Use the plus and minus icons on the left to navigate.
+rendering.macro.exitPages.name=Exit Pages
```

Comment:
You're missing translations for the macro parameters (``startDate`` and ``endDate``).